src/structured-data/structured-data-tags.js

```diff
@@ -33,18 +33,18 @@ function ldJson(type, fields) {
   };
 }
 
-function imageUrl(publisherConfig, s3Key) {
+function imageUrl(publisherConfig, s3Key, pageType) {
   const imageSrc = /^https?.*/.test(publisherConfig['cdn-image']) ? publisherConfig['cdn-image'] : `https://${publisherConfig['cdn-image']}`;
-  return `${imageSrc}/${s3Key}?w=480&h=270&auto=format%2Ccompress&fit=max`;
+  return pageType === 'story-page-amp' ? `${imageSrc}/${s3Key}?w=1200&h=750&auto=format%2Ccompress&fit=max`: `${imageSrc}/${s3Key}?w=480&h=270&auto=format%2Ccompress&fit=max`;
```

Comment:
pageType is something this function should not worry about. Can you accept width and height as optional params instead?